### PR TITLE
be and hpa suppport,clear resource,architecture arrange

### PR DESCRIPTION
**Pull request by shileifu**

### CODE REVIEW

**Review by kevincai — APPROVED** [review]

File: script/fe_prestop.sh [hunk 1]
```diff
@@ -3,3 +3,5 @@
 STARROCKS_ROOT=${STARROCKS_ROOT:-"/opt/starrocks"}
 STARROCKS_HOME=${STARROCKS_ROOT}/fe
 $STARROCKS_HOME/bin/stop_fe.sh
+
+
```
Comment (DorianZheng) [inline script/fe_prestop.sh:7]: remove duplicate blank line

File: script/be_prestop.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+STARROCKS_ROOT=${STARROCKS_ROOT:-"/opt/starrocks"}
+STARROCKS_HOME=${STARROCKS_ROOT}/be
+
+# graceful stop cn
```
Comment (DorianZheng) [inline script/be_prestop.sh:6]: graceful stop be

File: script/cn_entrypoint.sh [hunk 3]
```diff
@@ -165,6 +165,6 @@ add_self $svc_name
 
 trap exit_clean SIGTERM
 
-log_stderr "run start_cn.sh"
 update_conf_from_configmap
+log_stderr "run start_cn.sh"
```
[the remark this answers is not in the document]
Comment (shileifu) [reply script/cn_entrypoint.sh:169]: the function name is log_stderr, the print position is stdout, and the name as log_stderr original intention is to debug the script.

File: script/fe_prestop.sh (hunk 1 above)
Comment (shileifu) [reply script/fe_prestop.sh:7]: done

File: script/be_prestop.sh (hunk 2 above)
Comment (shileifu) [reply script/be_prestop.sh:6]: done

**Pull request merged**